enable/scrolled.py

```diff
@@ -535,8 +539,8 @@ def _handle_vertical_scroll(self, position):
                     view_position=[viewport.view_position[0], position]
                 )
 
-    def _mouse_thumb_changed(self, object, attrname, event):
-        if event == "down" and not self.continuous_drag_update:
+    def _mouse_thumb_changed(self, event):
```

Comment:
This is confusing code, this looks like a magic named method for when a trait `mouse_thumb` changes, however this class does not have a `mouse_thumb` trait!  It is only used as a dynamically hooked up handler (see above).

This is confusing because if it were a magically named method the signature would be "_name_changed(name, old, new)" whereas when it is just an otc handler it is "handler(object, name, new)".  
This specifically isn't relevant for observe now, but this practice is one we probably want to avoid as it is confusing.

enable has lots of uses of these magically named methods, but it appears sometimes they get hooked up with on_trait_change in addition to already listening because of the traits magic for their name

Reply:
I think these should be renamed where we know it's safe to do so. Like if a trait with the name "mouse_thumb" doesn't exist anywhere in Enable or Chaco, then we could be pretty sure that a rename was safe here.